src/algebra/group_with_zero/basic.lean

```diff
@@ -770,7 +765,7 @@ lemma div_eq_inv_mul : a / b = b⁻¹ * a :=
 by rw [div_eq_mul_inv, mul_comm]
 
 lemma mul_div_right_comm (a b c : G₀) : (a * b) / c = (a / c) * b :=
-by rw [div_eq_mul_inv, mul_assoc, mul_comm b, ← mul_assoc, div_eq_mul_inv]
+by { rw [div_eq_mul_inv, mul_assoc, mul_comm b, ← mul_assoc, div_eq_mul_inv] }
```

Comment:
Why would you change this?

Reply:
I probably added an extra tactic call and forgot to delete the brackets when I deleted the tactic.
```suggestion
by rw [div_eq_mul_inv, mul_assoc, mul_comm b, ← mul_assoc, div_eq_mul_inv]
```